src/index.ts

```diff
@@ -27,8 +31,16 @@ function setupOpenMRS() {
   return {
     lifecycle: getAsyncLifecycle(() => import("./root.component"), options),
     activate: (location: Location) =>
-      !location.pathname.startsWith(window.getOpenmrsSpaBase() + "login")
+      !location.pathname.startsWith(window.getOpenmrsSpaBase() + "login"),
+
+    extensions: [
```

Comment:
This extension should be offered by the `openmrs-esm-patient-registration` repository / package.